protocols/relay/examples/relay.rs

```diff
@@ -24,34 +24,44 @@
 //! relay client listening via the relay server and (3) a dialing relay client
 //! dialing the listening relay client via the relay server.
 //!
-//! 1. To start the relay server, run `cargo run --example=relay --package=libp2p-relay --mode relay --secret-key-seed 1 --address /ip4/<ip address>/tcp/<port>`.
-//!    The `-secret-key-seed` helps create a static peer id using the given number argument as  a seed.
-//!    The mode specifies whether the node should run as a relay server, a listening client or a dialing client.
-//!    The address specifies a static address. Usually it will be some loop back address such as `/ip4/0.0.0.0/tcp/4444`.
-//!    Example:
-//!    `cargo run --example=relay --package=libp2p-relay -- --mode relay --secret-key-seed 1 --address /ip4/0.0.0.0/tcp/4444`
-//!    `cargo run --example=relay --package=libp2p-relay -- --mode relay --secret-key-seed 1 --address /ip6/::/tcp/4444`
+//! 1. To start the relay server, run `cargo run --example=relay --package=libp2p-relay --mode relay
+//! --secret-key-seed 1 --address /ip4/<ip address>/tcp/<port>`.    The `-secret-key-seed` helps
+//! create a static peer id using the given number argument as  a seed.    The mode specifies
+//! whether the node should run as a relay server, a listening client or a dialing client.
+//!    The address specifies a static address. Usually it will be some loop back address such as
+//! `/ip4/0.0.0.0/tcp/4444`.    Example:
```

Comment:
While maybe not properly formatted before, I don't think this is either.

Reply:
Fair enough, I'll look into it and see if I can make things work somehow.